include/spdlog/fmt/bundled/format.h

```diff
@@ -2897,6 +2899,8 @@ FMT_CONSTEXPR const Char* parse_format_specs(const Char* begin, const Char* end,
     handler.on_space();
     ++begin;
     break;
+  default:
+    break;
```

Comment:
this is bundled copy from the fmt project, so should be fixed there

Reply:
I'll fix it in the upstream fmtlib.